polymorphism/README.md

```
@@ -0,0 +1,192 @@
+In classic C++, interfaces are defined via [abstract base classes](https://en.cppreference.com/w/cpp/language/abstract_class) from which the implementation (publicly) inherits. In this example, we have defined such an abstract class:
+
+```c++
+class ISuperCoolFeatures {
+public:
+    virtual std::string coolFeature() const = 0; // Pure virtual function
+    virtual void set(std::string s) = 0; // Pure virtual function
+    virtual ~ISuperCoolFeatures() = default; // Virtual destructor
+};
+```
+
+Instead of defining the member function body (which is possible in a regular base class), the member functions of an abstract interface class are declared "pure virtual" by adding `= 0` to their declaration. 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)